core/storage/trie/polkadot_trie/polkadot_trie_impl.cpp

```diff
@@ -43,7 +43,8 @@ namespace kagome::storage::trie {
     return root_;
   }
 
-  outcome::result<void> PolkadotTrieImpl::put(const Buffer &key, Buffer &&value) {
+  outcome::result<void> PolkadotTrieImpl::put(const Buffer &key,
+                                              Buffer &&value) {
```

Comment:
Is it really required to use &&?

Reply:
This way gives us ability to use rvalue and lvalue and avoid redundant copy-ctor call.

Reply:
That's it.